utils/build/docker/python/flask/app.py

```diff
@@ -428,6 +425,34 @@ def delivery_report(err, msg):
     return response
 
 
+@app.route("/dsm/inject")
+def inject_dsm_context():
+    topic = flask_request.args.get("topic")
+    integration = flask_request.args.get("integration")
+    headers = {}
+
+    reset_dsm_context()
+
+    ctx = data_streams_processor().set_checkpoint(["direction:out", "topic:" + topic, "type:" + integration])
+    DsmPathwayCodec.encode(ctx, headers)
+
+    return Response(json.dumps(headers))
+
+
+@app.route("/dsm/extract")
```

Comment:
could you document those new endpoint in the doc ?

Reply:
done